src/devices/hubled.ts

```diff
@@ -75,10 +69,24 @@ export class HubLED extends Device {
 }
 
 export namespace HubLED {
-
-    export enum Mode {
-        COLOR = 0x00,
-        RGB = 0x01
-    }
-
+    export const modes: { [name: string]: IDeviceMode } = {
+        COLOR: {
+            input: false,
+            num: {
+                [HubType.MOVE_HUB]: 0x00,
+                [HubType.TECHNIC_MEDIUM_HUB]: 0x00,
+                [HubType.HUB]: 0x00,
+                [HubType.WEDO2_SMART_HUB]: 0x00
```

Comment:
I do this already in a few places - for example in `currentsensor.ts` there is a default set for `HubType.UNKNOWN`. Perhaps this could follow the same model.

Reply:
It may be weird but I don't like using `UNKNOW` as the `ALL` placeholder, it may lead to misunderstanding while reading the code.